tests/composite_pk/test_values.py

```diff
@@ -128,18 +128,18 @@ def test_values_list(self):
             self.assertSequenceEqual(
                 User.objects.values_list("pk", "pk").order_by("pk"),
                 (
-                    (self.user_1.pk,),
-                    (self.user_2.pk,),
-                    (self.user_3.pk,),
+                    (self.user_1.pk, self.user_1.pk),
+                    (self.user_2.pk, self.user_2.pk),
+                    (self.user_3.pk, self.user_3.pk),
```

Comment:
Interesting that we tested the opposite for composite primary key in the small amount of time after 65ad4ade74dc9208b9d686a451cd6045df0c9c3a landed.